Sources/Auth/Services/App/AppRespondSubscriber.swift

```diff
@@ -41,7 +38,8 @@ class AppRespondSubscriber {
         networkingInteractor.responseSubscription(on: AuthRequestProtocolMethod())
             .sink { [unowned self] (payload: ResponseSubscriptionPayload<AuthRequestParams, Cacao>)  in
 
-                activatePairingIfNeeded(id: payload.id)
+                //TODO - call pairing client to activate
```

Comment:
issue to not forgot?

Reply:
I will prepare an epic for Pairing tomorrow.